sdk/azcore/runtime/request.go

```diff
@@ -148,6 +153,38 @@ func SetMultipartFormData(req *policy.Request, formData map[string]any) error {
 		return nil
 	}
 
+	writeMultipartContent := func(fieldname, filename string, mpc streaming.MultipartContent) error {
```

Comment:
Why is this local but not `quoteEscaper`?

Reply:
Good point, moved `quoteEscaper`.